lib/validators/baseRef.js

```diff
@@ -0,0 +1,43 @@
+const { Validator } = require('./validator')
+
+class BaseRef extends Validator {
+  constructor () {
+    super('baseRef')
+    this.supportedEvents = [
+      'pull_request.opened',
+      'pull_request.edited',
+      'pull_request_review.submitted',
+      'pull_request_review.edited',
+      'pull_request_review.dismissed',
+      'pull_request.labeled',
+      'pull_request.milestoned',
+      'pull_request.demilestoned',
+      'pull_request.assigned',
+      'pull_request.unassigned',
+      'pull_request.unlabeled',
+      'pull_request.synchronize',
+      'pull_request.push_synchronize'
+    ]
```

Comment:
When all the actions of an event is supported we typically like to write this like so:
```js
this.supportedEvents = [
      'pull_request.*',
      'pull_request_review.*'
    ]
```

This way, it is understood that everything within pull_request and pull_request_review is supported. It is becomes consistent with the docs.

Reply:
@jusx makes sense, I changed it.